extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/pages/asset/services/utils/AssetUtils.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright (c) 2023 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.ext.wrapper.api.ui.pages.asset.services.utils;
+
+import de.sovity.edc.ext.wrapper.api.ui.model.DataAddressDto;
+import lombok.RequiredArgsConstructor;
+import org.eclipse.edc.spi.types.domain.DataAddress;
+
+@RequiredArgsConstructor
+public class AssetUtils {
```

Comment:
real name would be DataAddressMapper

A class "AssetUtils" would imply that it is mostly methods that take an asset as a first parameter